lstchain/scripts/lstchain_mc_sensitivity.py

```diff
@@ -82,67 +90,61 @@ def main():
     gammas_mc = dl2[dl2.mc_type == 0]
     protons_mc = dl2[dl2.mc_type == 101]
     good_gammas = dl2
-
-    sns.distplot(gammas_mc.gammaness, label='gammas')
-    sns.distplot(protons_mc.gammaness, label='protons')
+    
+    sns.distplot(gammas_mc.gammaness, label='gammas');
+    sns.distplot(protons_mc.gammaness, label='protons');
     plt.legend()
     plt.tight_layout()
     plt.show()
-    sns.distplot(gammas_mc.mc_energy, label='gammas')
-    sns.distplot(protons_mc.mc_energy, label='protons')
+    sns.distplot(gammas_mc.mc_energy, label='gammas');
+    sns.distplot(protons_mc.mc_energy, label='protons');
     plt.legend()
     plt.tight_layout()
     plt.show()
-    sns.distplot(gammas_mc.log_reco_energy, label='gammas')
-    sns.distplot(protons_mc.log_reco_energy, label='protons')
-    plt.legend()
+    sns.distplot(gammas_mc.reco_energy, label='gammas');
+    sns.distplot(protons_mc.reco_energy, label='protons');
```

Comment:
are you sure you want to plot the distribution without log? it won't bring much info.
I'd suggest
`gammas_mc.reco_energy.apply(np.log10)`

Reply:
Do not remember why this was here but ok, done